### ISSUE TITLE
Cannot connect to R kernel

### ISSUE THREAD

**Issue by pgensler** [jupyterlab/jupyterlab#3892]
When I try to connect to the R Kernel, I get the following:
```

[I 19:38:46.519 LabApp] Kernel started: 4e2eceb3-d8f4-45e6-9a75-91308b6bcb1e
Error in loadNamespace(name) : there is no package called ‘IRkernel’
Calls: :: ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted

```

I just tried to install the latest version of Anaconda, and maybe it is missing the latest version of the` IRkernel` package?
It sounds like the Anaconda Distribution does not have the `IRkernel` installed properly?

Jupyter Lab Version 0.31.5
`python --version` is `Python 3.6.4 :: Anaconda, Inc.`
Let me know if you need more info, happy to provide.

**Comment by blink1073** [comment]
Hi @pgensler, I don't believe the Anaconda distribution comes with the `IRKernel` package by default.   You should be able to get it as `conda install -c r r-irkernel`

**Comment by pgensler** [comment]
Interesting, I did not know you could install `irkernel` through conda. Is this specified within the docs? I would think that this should be in Anaconda. I'll try and see if that works.

**Comment by saulshanabrook** [comment]
Feel free to comment if you would like this reopened.

**Comment by pgensler** [comment]
@saulshanabrook this is probably feedback more for Anaconda, but if you want a user to use R within JupyterLab, I'd auto-install the IRKernel package, so the user is not forced to deal with figuring out what virtual environment needs what package to get up and running. Does that make sense?

**Comment by saulshanabrook** [comment]
Yeah that makes sense. We could depend on the R kernel in the conda package. 

I'll reopen this for documenting how to install other kernels.

**Comment by MichaelChirico** [comment]
Just for the record, I installed non-`conda` R kernel also (`python` is also non-`conda`):

```
apt-get update && apt-get install r-base
Rscript -e "install.packages('IRkernel'); IRkernel::installspec()"
```

It took a minute or two, but eventually the "New notebook" page detected the new R kernel
